### PR TITLE
kimberliteBidAdapter: video media type support

### PR DESCRIPTION
**Pull request by os-solta**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter  
- [x] Updated bidder adapter https://github.com/prebid/prebid.github.io/pull/5504
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added video media type to kimberliteBidAdapter

### CODE REVIEW

**Comment by os-solta** [comment]
Hi! I'm not sure about the reason of CI build fail. Could someone help me with the reason please?

**Comment by patmmccann** [comment]
Just a flaky test, no worries

File: modules/kimberliteBidAdapter.js [hunk 1]
```diff
@@ -34,19 +34,40 @@ const converter = ortbConverter({
   imp (buildImp, bidRequest, context) {
     const imp = buildImp(bidRequest, context);
     imp.tagid = bidRequest.params.placementId;
+    const mediaTypes = bidRequest.mediaTypes;
+    if (imp[VIDEO]) {
+      deepSetValue(imp, 'video.linearity', mediaTypes[VIDEO].linearity || 1);
+      deepSetValue(imp, 'video.protocols', mediaTypes[VIDEO].protocols || [3, 6]);
+      deepSetValue(imp, 'video.placement', mediaTypes[VIDEO].placement || 1);
```
Comment (patmmccann) [inline modules/kimberliteBidAdapter.js:41]: I think the ortb convertor gives you all these for free, no need to copy them. Is that correct @dgirardi ?

Comment (dgirardi) [reply modules/kimberliteBidAdapter.js:41]: I think this is to set default values, which the library does not provide.

Comment (patmmccann) [reply modules/kimberliteBidAdapter.js:41]: Ah I see @os-solta these defaults aren't good guesses, you're better off requiring the publisher make a choice

**Review by patmmccann — CHANGES_REQUESTED** [review]
Video defaults on placement and startdelay

File: modules/kimberliteBidAdapter.js (hunk 1 above)
Comment (os-solta) [reply modules/kimberliteBidAdapter.js:41]: Looks like i'll just skip these defaults here, thanks.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**